vars/signArtifacts.groovy

```diff
@@ -24,11 +34,19 @@ void call(Map args = [:]) {
             export UNSIGNED_BUCKET=${SIGNER_CLIENT_UNSIGNED_BUCKET}
             export SIGNED_BUCKET=${SIGNER_CLIENT_SIGNED_BUCKET}
 
-            $WORKSPACE/sign.sh ${args.artifactPath} --sigtype=${args.signatureType} --component=${args.component} --type=${args.type}
+            $WORKSPACE/sign.sh ${arguments}
         """
     }
 }
 
+String generateArguments(args) {
+    // artifactPath is mandatory and the first argument
+    String arguments = args.artifactPath
```

Comment:
Make a copy of `args` without `artifactsPath`, then iterate through the rest.

Reply:
I updated the method that iterates through the `args` with `artifactsPath` removed.